core/src/main/java/greencity/controller/UserProfileController.java

```diff
@@ -35,13 +35,14 @@ public class UserProfileController {
     @ApiOperation(value = "Update user profile")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK, response = UserProfileDto.class),
+        @ApiResponse(code = 204, message = HttpStatuses.NO_CONTENT),
         @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
         @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
     })
     @PutMapping("/user/update")
     public ResponseEntity<UserProfileUpdateDto> updateUserData(@ApiIgnore @CurrentUserUuid String userUuid,
         @Valid @RequestBody UserProfileUpdateDto userProfileUpdateDto) {
-        return ResponseEntity.status(HttpStatus.OK)
+        return ResponseEntity.status(HttpStatus.NO_CONTENT)
             .body(ubsClientService.updateProfileData(userUuid, userProfileUpdateDto));
```

Comment:
Are you sure we need to return 204 code, not 200? As I read, No Content answer should be sent only when we don’t response any information, but here we return UserProfileUpdateDto

Reply:
I was also bothered that No Content answer should be sent only when we don’t response any information. But it was clearly stated in the task that when we edit data in user profile and send request for update response 204 is expected

Reply:
@innayashna I do not agree with this issue. As we return content, we should set up status code 200. Please write a comment on the issue that it's not a relevant task because the content will return.